### DISCUSSION TITLE
How do I edit or delete media?

### DISCUSSION THREAD

**Discussion by TheBig-O** [IgnisDa/ryot#722] [Q&A] [answered]
After adding an audiobook, I realize that the information I used is incorrect. However, I cannot find any method of editing or deleting the erroneous entry. Is there a menu setting I'm missing? How can I make edits to exiting items?

**Comment by IgnisDa** [comment]
There is no way currently, though I plan to add one. Would it be possible for you to do that directly through SQL queries?

**Reply by TheBig-O** [reply]
Yes, I'm comfortable with sql and could easily do the updates if I knew the tables to work with.

**Reply by IgnisDa** [reply, marked as answer]
Please take a look at the `metadata` table in the postgres. Most columns are self explanatory.
